src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -149,7 +152,7 @@ public Drivetrain(ShuffleboardTab drivetrainTab, ShuffleboardTab swerveModulesTa
     m_poseEstimator.setVisionMeasurementStdDevs(VisionConstants.kBaseVisionPoseStdDevs);
 
     setPigeonYaw(DriveConstants.kStartingHeadingDegrees);
-
+    
```

Comment:
Remove random spaces, I know it is knit picky but it is annoying for a few reasons. It make the file change view larger than it needs to be. It makes random files appear in the pull request. Random spaces can add up and It makes random gaps of code which increases the distance to scroll though code. Everyone dose it but at least clean it up before you make a PR. So I know it's knit picky but it makes is easier for the people reviewing and working on the code.